forms-flow-components/src/components/CustomComponents/ImportModal.tsx

```diff
@@ -76,20 +76,23 @@ export const ImportModal: React.FC<ImportModalProps> = React.memo(
     const redColor = computedStyle.getPropertyValue("--ff-red-000");
     const [selectedFile, setSelectedFile] = useState<File | null>(null);
     const [uploadProgress, setUploadProgress] = useState(0);
-    const skipImport = "Skip, do not import";
+    const hasVersion = (item) => item?.majorVersion != null || item?.minorVersion != null;
+    const skipImport = fileItems && Object.values(fileItems).some(hasVersion) 
+    ? "Skip, do not import" 
+    : "";
     const [selectedLayoutVersion, setSelectedLayoutVersion] = useState<{
       value: any;
       label: string;
     } | null>({
       value: true,
-      label: skipImport,
```

Comment:
keep it same as previous instead. The constant string was added to avoid adding the same string twice here.